src/lib/deployWorkers.ts

```diff
@@ -778,7 +790,8 @@ async function resolveWorker({
         )}`,
       );
 
-      const { overrideModules, serviceConfig } = maybeServiceConfig;
+      const { overrideModules, serviceConfig, totalMemoryLimit } =
+        maybeServiceConfig;
```

Comment:
```suggestion
        serviceConfig;
```
I still think that it's better to let ts config decide whether it's maybe config or no maybe. I don't feel like it's a good practise to put type related things in var name

Reply:
I more or less try to remove these maybe things where they are not required, though there a benifits to have them sometimes. But right here not so straightforward, cause there is actually a serviceConfig inside maybeServiceConfig, where serviceConfig in the actual config from file and maybeServiceConfig in this case is just a thing that contains the actual config + overrides for modules + memory limit. Don't really kno,w how to rename that. We can leave it as it is or think of some why to rename stuff here, can't come up with a satisfying name

Reply:
Maybe `combinedConfig` or `configWithOverrides` ?